### ISSUE TITLE
Accessibility: Plugin single missing focus indicator on languages button

### ISSUE THREAD

**Issue by amberhinds** [WordPress/wordpress.org#264]
On the plugin single page ([Gutenberg plugin test page](https://test.wordpress.org/plugins/gutenberg/)), there are styles removing the outline from the languages button, which cause it to not have a focus outline when receiving keyboard focus.

This is the element:
<img width="473" alt="Screenshot 2024-04-16 at 5 29 57 PM" src="https://github.com/WordPress/wordpress.org/assets/13153456/118850f9-7cb4-435f-954d-64d87529590e">

This is the CSS causing interference:
```
.button-link {
    background: none;
    border: 0;
    border-radius: 0;
    box-shadow: none;
    cursor: pointer;
    margin: 0;
    outline: none;
    padding: 0;
}

```

Here's a quick video showing the missing outline on focus.

https://github.com/WordPress/wordpress.org/assets/13153456/8f5da263-9ab9-421f-bda4-8bfabf32df07
